inscriptions/ISic003418.xml

```diff
@@ -183,17 +180,24 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI178075">
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#VM">
                 <ab>
-                    <lb n="1"/>
-
-
-                    Εἰρήνα νύμφη<lb n="2"/>ὧδε κεῖται.<lb n="3"/>κατὰ τοῦ μυστηρί-<lb n="4"/>ου οὖν τούτου μὴ τ-<lb n="5"/>ὶς ὧδε ἀνύξῃ.
-                    <lb n="6"/>
+                    <lb n="1"/>Εἰρῆνα Νύμφη
+                    <lb n="2"/>ὧδε κεῖται
+                    <lb n="3"/>κατὰ τοῦ μυστηρί
+                    <lb n="4" break="no"/>ου οὖν τούτου μη
+                    <lb n="5" break="no"/><choice><reg>δ</reg><orig>τ</orig></choice>ὶς ὧδε ἀνύξη
                 </ab>
             </div>
             <div type="apparatus">
-                <p><!--external apparatus criticus (if applicable)--></p>
+                <listApp>
+                    <app>
+                        <note>Text after Orsi 1900</note>
+                    </app>
+                    <app loc="4-5">
+                        <note>μηδεὶς ἀνοίξη</note> <!-- should we keep the regularised form here and add in the text with <choice> only the minor changes (I have always tried to leave the text as it appears on the stone by inserting <choice>etc. in case of normalisations/errors)? does this create problems for future lemmatisation? -->
```

Comment:
I firmly believe we should encode all such cases with choice/reg/orig (and NOT sic/corr). I think the whole point about choice is that we can still lemmatise the word independent of this, and indeed it then has the great merit that the word will be found by searching for the 'normal' form, but one can also run searches that owuld pull up all variations from the normal form (i.e. cases where choice occurred, etc.).